### DISCUSSION TITLE
支付宝已付款续期1年，使用期限没有延长1年,问题3周都没有解决，请退款！

### DISCUSSION THREAD

**Discussion by aladdindune** [getlantern/lantern-forum-cn#975] [General]
![1727837607809](https://github.com/user-attachments/assets/952367d7-541a-4af6-b9bd-3b647ce4fcfc)
![1727837607818](https://github.com/user-attachments/assets/1ecae2b0-f6d0-4551-9620-2f3bddc0ee2d)

**Comment by MaxMKMax** [comment]
@aladdindune 您好. 请您提供我账号邮箱.

**Comment by aladdindune** [comment]
***@***.***。

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by aladdindune** [comment]
这是上次的回复。

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
